### DISCUSSION TITLE
Reactor Kafka examples

### DISCUSSION THREAD

**Discussion by almogtavor** [hantsy/spring-reactive-sample#260] [Ideas]
It would be great if Kafka samples would be added. This is a pretty common use case that doesn't have lots of examples. Especially when combined with libraries like Spring Data.

**Comment by hantsy** [comment]
Great idea, I have created a task for myself #261, I will create an example to demonstrate Reactor Kafka/Spring WebFlux ASAP.

**Comment by almogtavor** [comment]
Actually I think it would be better if the example would be more of a data pipeline kind. The Webflux usage with Kafka is pretty obvious into my opinion. I suggest an application that consumes a Kafka topic, creates batches, and writes them reactively to Mongo for example.

For example you can see https://tech.trivago.com/2021/03/16/java-reactive-programming-effective-usage-in-a-real-world-application/

**Comment by hantsy** [comment]
Added reactor Kafka and RabbitMQ, https://github.com/hantsy/spring-reactive-sample/tree/master/boot-reactor-kafka
